mifosng-provider/src/main/java/org/mifosplatform/organisation/staff/serialization/StaffCommandFromApiJsonDeserializer.java

```diff
@@ -134,6 +148,17 @@ public void validateForUpdate(final String json) {
 
         if (this.fromApiJsonHelper.parameterExists("isActive", element)) {
             final Boolean activeFlag = this.fromApiJsonHelper.extractBooleanNamed("isActive", element);
+            //Need to add here check to see if any clients and loans are assigned to this staff if staff is being set to inactive --LJB
+            final Boolean forceStatus = this.fromApiJsonHelper.extractBooleanNamed("forceStatus", element);
+            if ((activeFlag == false && forceStatus == null) || 
+                (activeFlag == false && forceStatus != true)) {
+                List<ClientData> clientData = (List<ClientData>) clientReadPlatformService.retrieveAllForLookup("staff_id = "+ staffId);
```

Comment:
Glad to do so.  I'll add the following validation. When setting staff to inactive, check that none of the entities listed above have a reference to the staff that is being set inactive as long as the entity itself is active.  I'll implement it with the same behavior as I did for clients.  Please let me know if this is not the behavior you are expecting.